cmd/crossplane/main.go

```diff
@@ -33,13 +35,17 @@ import (
 	"github.com/crossplane/crossplane/apis"
 	"github.com/crossplane/crossplane/cmd/crossplane/core"
 	"github.com/crossplane/crossplane/cmd/crossplane/rbac"
+	"github.com/crossplane/crossplane/internal/version"
 )
 
 type debugFlag bool
+type versionFlag bool
 
 var cli struct {
 	Debug debugFlag `short:"d" help:"Print verbose logging statements."`
 
+	Version versionFlag `name:"version" help:"Print version and quit." short:"v"`
```

Comment:
small nit: having the `short` field listed first, similar to the `debugFlag`, may make it easier and more consistent to read.

Reply:
Done!